insta-ui/src/app/server/llm/types.ts

```diff
@@ -0,0 +1,11 @@
+// app/server/llm/types.ts
+export interface Message {
+    role: 'system' | 'user' | 'assistant';
+    content: string;
+}
```

Comment:
This looks duplicated with the following. Feel free to directly use that one to avoid duplication.

https://github.com/junkaiman/ui-generator/blob/ce2e67dcce2bda2be2d885f66a0793c31083bc69/insta-ui/src/lib/types.ts#L17-L20